ios/PolyPodApp/PolyPod/AppDelegate.swift

```diff
@@ -5,7 +5,7 @@ import CoreData
 @main
 class AppDelegate: UIResponder, UIApplicationDelegate {
     private static let updateNotificationCheckIdentifier = "coop.polypoly.polypod.updateNotificationCheck"
-    
+
```

Comment:
This is interesting an interesting one: For me, Xcode always insists on adding trailing whitespace to empty lines. Since that's what _control + i_ does, I figured we can at least keep the code base consistent that way. If you mark the new code you added and hit _control + i_, does it also add this trailing whitespace?

I'm _very_ onboard with getting rid of this, but didn't dare to fight with Xcode's formatting settings (and figuring out how to store these changes in the repository) yet. And I'd definitely want to do that in a separate PR :D Would have to modify pretty much every empty line we have in the code base.

Reply:
Yes Xcode can be a little bit nasty on that one, from my experience it is usually advised to remove such spaces, as it might happen to have merging conflicts with empty lines(I now it sounds wild :D). It didn't happen to me, so I guess we might leave them as is.

Reply:
Yeah, I think for now just running `CTRL + i` on our changes is a decent solution to avoid any such conflicts. They do come from unrelated changes, like the line I'm commenting on here :D